README.md

```diff
@@ -5,6 +5,8 @@ The previously used vision document is superseeded by the continous, changing vi
 
 ## [Browse project ideas](https://github.com/dlang/projects/issues)
 
+<br>
```

Comment:
Why this?

Reply:
Sorry for mixing this change, but I thought this looks clearer and makes it clear that the issue tracker is the main link.

Before:

![image](https://user-images.githubusercontent.com/4370550/57639459-42632b00-75b0-11e9-8c04-0135b4cebb16.png)


After:

![image](https://user-images.githubusercontent.com/4370550/57639528-63c41700-75b0-11e9-9163-ad9a183cc72e.png)

Reply:
I think it looks weird, but whatever.

Reply:
Removed it from this PR.